src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.test.js

```diff
@@ -61,6 +61,7 @@ const defaultAjaxImpl = {
 };
 
 const verifyEnabled = (item) => expect(item).not.toHaveAttribute('disabled');
+const verifyDisabled = (item) => expect(item).toHaveAttribute('disabled');
```

Comment:
Would [toBeDisabled](https://github.com/testing-library/jest-dom?tab=readme-ov-file#tobedisabled) work here?

```suggestion
const verifyDisabled = (item) => expect(item).toBeDisabled();
```